src/sap.ui.core/test/sap/ui/core/qunit/ExpressionParser.qunit.js

```diff
@@ -43,8 +43,9 @@ sap.ui.require([
 	 * @param {string} sTitle - test title
 	 * @param {object[]} aFixtures - the array of test fixtures
 	 * @param {function} [fnInit] - optional initializer function to e.g. set stubs
+	 * @param {function} [fnTearDown] optional tear down function to e.g. destroy stubs
```

Comment:
Written as is, the jsdoc implies that one can pass fnTearDown without an fnInit. In that case, the tear down will be called at initialization.